modules/miscutil/lib/inveniocfg.py

```diff
@@ -879,21 +885,25 @@ def cli_cmd_load_demo_records(conf):
                 "%s/bin/bibupload 1" % CFG_PREFIX,
                 "%s/bin/bibdocfile --textify --with-ocr --recid 97" % CFG_PREFIX,
                 "%s/bin/bibdocfile --textify --all" % CFG_PREFIX,
-                "%s/bin/bibindex -u admin" % CFG_PREFIX,
+                "%s/bin/bibindex -u admin -w collection" % CFG_PREFIX,
                 "%s/bin/bibindex 2" % CFG_PREFIX,
-                "%s/bin/bibindex -u admin -w global" % CFG_PREFIX,
-                "%s/bin/bibindex 3" % CFG_PREFIX,
-                "%s/bin/bibreformat -u admin -o HB" % CFG_PREFIX,
-                "%s/bin/bibreformat 4" % CFG_PREFIX,
                 "%s/bin/webcoll -u admin" % CFG_PREFIX,
-                "%s/bin/webcoll 5" % CFG_PREFIX,
+                "%s/bin/webcoll 3" % CFG_PREFIX,
+                "%s/bin/bibindex -u admin" % CFG_PREFIX,
+                "%s/bin/bibindex 4" % CFG_PREFIX,
+                "%s/bin/bibindex -u admin -l" % CFG_PREFIX,
+                "%s/bin/bibindex 5" % CFG_PREFIX,
+                "%s/bin/bibreformat -u admin -o HB" % CFG_PREFIX,
+                "%s/bin/bibreformat 6" % CFG_PREFIX,
                 "%s/bin/bibrank -u admin" % CFG_PREFIX,
-                "%s/bin/bibrank 6" % CFG_PREFIX,
+                "%s/bin/bibrank 7" % CFG_PREFIX,
                 "%s/bin/bibsort -u admin -R" % CFG_PREFIX,
-                "%s/bin/bibsort 7" % CFG_PREFIX,
+                "%s/bin/bibsort 8" % CFG_PREFIX,
                 "%s/bin/oairepositoryupdater -u admin" % CFG_PREFIX,
-                "%s/bin/oairepositoryupdater 8" % CFG_PREFIX,
-                "%s/bin/bibupload 9" % CFG_PREFIX,]:
+                "%s/bin/oairepositoryupdater 9" % CFG_PREFIX,
+                "%s/bin/bibupload 10" % CFG_PREFIX,
+                "%s/bin/bibauthorid -u admin --update-personid" % CFG_PREFIX,
```

Comment:
`bibauthorid` shouldn't really be plugged in here?  See https://github.com/inveniosoftware/invenio/issues/1862